packages/consensus/src/types/chain_state.cairo

```diff
@@ -34,6 +34,8 @@ pub struct ChainState {
     /// it's possible that one block could have an earlier timestamp
     /// than a block that came before it in the chain.
     pub prev_timestamps: Span<u32>,
+    /// Median Time Past (MTP) of the current block
+    pub median_time_past: u32,
```

Comment:
No need to add this as chain state field since it is a derivative of prev_timestamps

Reply:
Okay.